src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@use './variables' as *;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__main-image {
+    margin: 32px auto 40px;
+  }
+
+  &__item--name {
+    margin-bottom: 4px;
+  }
+
+  &__item--description {
+    margin-bottom: $margin-main;
+  }
+
+  &__wrapper {
+    margin-bottom: $margin-main;
+  }
+
+  &__button {
+    margin-bottom: $margin-main;
+  }
+}
```

Comment:
There's an inconsistency in the spacing of the padding property. The SCSS style guide recommends using consistent units for padding and margin. Suggest using variables for spacing values to maintain consistency and ease of maintenance.